docker/entrypoint.sh

```diff
@@ -1,9 +1,15 @@
 #! /bin/bash
 cd lnprototest || exit
 
+if [ "$LN_IMPL" == "ldk" ]; then
+  RUNNER="ldk_lnprototest.Runner"
+elif [ "$LN_IMPL" == "clightning" ]; then
+  RUNNER="lnprototest.clightning.Runner"
+fi
+
```

Comment:
If we apply this suggestion, https://github.com/rustyrussell/lnprototest/pull/133/files#r2008741401 I think the script will be much simpler?